src/pep8ext_naming.py

```diff
@@ -375,7 +357,7 @@ def visit_functiondef(self, node, parents, ignore=None):
         def arg_name(arg):
             try:
                 return arg, arg.arg
-            except AttributeError:  # PY2
+            except AttributeError:
```

Comment:
This also handles the `arg is None` case. We could make that explicit here instead of catching `AttributeError`, though.